backend/order/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
+    path('', views.ListAllRequisitions.as_view(), name="list_order"),
```

Comment:
Para aplicar la otra sugerencia

```suggestion
    path('', views.ListOrders.as_view(), name="list_order"),
```